src/blocklyc.js

```diff
@@ -751,15 +722,15 @@ function serial_console() {
                         pTerm.display(e.data);
                     }
                 }
-                $('#serial_console').focus();
+                pTerm.focus();
```

Comment:
Added a function to the propTerm class so that the containing element would not need to be referenced anywhere other than the initializer for the class.